packages/teams-js/src/internal/validOrigins.ts

```diff
@@ -34,7 +35,11 @@ async function getValidOriginsListFromCDN(): Promise<string[]> {
         });
       })
       .catch((e) => {
-        validateOriginLogger('validOrigins fetch call to CDN failed with error: %s. Defaulting to fallback list', e);
+        if (e.name === 'TimeoutError') {
+          validateOriginLogger('validOrigins fetch call to CDN failed due to Timeout. Defaulting to fallback list', e);
```

Comment:
>, e [](http://example.com/codeflow?start=114&length=3)

Does this do anything without the `%s` in the string?

Reply:
Nope, good catch. I've removed it